File: Arrays/Maximum and minimum elements in an array/MaxAndMin.java
// to find the maximum and minimum value in an array
import java.util.*;

public class MaxAndMin
{
        public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("enter size of array: ");
        int n = sc.nextInt();
        int[] arr = new int[n];

        System.out.println("enter elements in the array: ");
        for(int i=0; i<n; i++){
            arr[i] = sc.nextInt();
        }

        int max = arr[0];
        int min = arr[0];
        for(int i=1; i<n; i++){
            if(arr[i]>max){
                max = arr[i];
            }
            if(arr[i] < min){
                min = arr[i];
            }
        }
        System.out.println("maximum: "+max);
        System.out.println("minimum: "+min);
    }
}

File: Arrays/Maximum and minimum elements in an array/MaxAndMin.py
# to print maximum and minimum value in a list
#here no need of specifying the size, you can directly enter the list elements with spaces
n = list(map(int, input().split()))
maximum = n[0] 
minimum = n[0]
print(n)
for i in range(1,len(n)):
    if(n[i] > maximum):
        maximum = n[i]
    if(n[i] < minimum):
        manimum = n[i]
print("maximum: ",maximum)
print("minimum: ",minimum)
